### ISSUE TITLE
Investigate path for migrating SCCs from source to destination cluster

### ISSUE THREAD

**Issue by dymurray** [fusor/mig-controller#35]
Currently Velero (and by extension the controller) will not migrate SCC profiles across cluster since these are cluster-scoped resources that don't have any link to Kube resources. As a first pass, we should provide documentation to the end user that they will need to manually migrate custom SCC profiles to the destination cluster prior to the migration of their applications.

Down the line we should add support in the UI for a user to select custom SCC profiles they wish to migrate and the controller should set the SCCs in the `IncludedResources` field.

**Comment by jwmatthews** [comment]
At a minimum we need to document a manual path.
I'd like to see what other options we could support.

Dylan suggested this may come down to:
 - Controller investigates a namespace and determines all the SCC's involved
 - Info on all SCCs used is communicated to user
 - User goes through a potential mapping of source_SCC to dest_SCC similar to StorageClass handling
 
We will need to think through cases of:
 - If SCC does not exist on destination, we recreate
 - If SCC does exist on destination, but it has different details than existing what do we do?  Do we alert of differences and allow to proceed, do we try to merge, do we restore as a different name, etc

**Comment by sseago** [comment]
So I just hit what seems to be another aspect of this issue. In trying to migrate stateful apps across (using mysql as an example) to test storageclass change, migrating to rbd/RWO is failing on destination, while migrating to cephfs/RWX is working.

**Comment by sseago** [comment]
I can narrow down the above slightly. When I migrated those failing RWO-mounted applications to cephfs rather than rbd (overriding the controller-chosen storageclass on the migplan), they all worked. So RWO vs. RWX doesn't appear to be relevant, but for some reason cephfs is not exhibiting the same failure behavior as I saw with rbd and dylan reported with whatever storageclass he was using (ebs/gp2?).
